Assets/Scripts/UIScripts/PauseScreen.cs

```diff
@@ -89,7 +93,7 @@ public void MainMenu()
     public void PauseOrResumeController()
     {
         if (GameManager.Current.HasGameEnded()) return;
-        if (GameManager.Current.IsGamePaused())
+        if (GameManager.Current.IsGamePaused() && settingsMenuUI.activeInHierarchy)
```

Comment:
Wait, wouldn't this change mean we can only resume the game if the game is paused and the settings menu is open? So if the settings menu is closed, we can't resume? Correct me if Im wrong lol

Reply:
This was put here bc I saw we did something similar above, and without it there was a countdown when I clicked pause on the controller lol. But it was working fine, I tested pausing/resuming without opening the settings again. If you want to test on controller that would be great, otherwise I can test again in the evening

Reply:
I just tested it using controller and I couldn't exit out of the pause menu by clicking "Start". And when I opened the settings menu and then clicked "Start", it got rid of the settings menu but the pause menu was still there (Idk why because the Resume() function sets it to inactive 🤔) and the countdown triggered.

This is what we did above: 
![image](https://user-images.githubusercontent.com/45300621/205145150-68ced9d6-052a-4060-af2b-829966cf2d53.png)

If the game is paused and the settings menu was not active, we resume.
If the game is paused and the settings menu was active, we exit the settings menu.
If the game is not paused, we resume

Reply:
Ok so I wanted to find out why the settings menu disappears, but the pause menu is still there (because the Resume() function sets both to inactive). Turns out, whenever I click the start button once, the PauseOrResumeController() method gets triggered 3 times lol:
![image](https://user-images.githubusercontent.com/45300621/205146397-d4c50119-dd5c-48d9-91bd-2d3aac79c4aa.png)

So, when I open the settings menu and click "Start", it triggers Resume() (which gets rid of the settings and pause menus) and then it triggers Pause() twice, which brings up the pause menu again lol:
![image](https://user-images.githubusercontent.com/45300621/205146763-83df39cf-67db-4e52-8812-fc58062daf07.png)

Reply:
This is def strange. Not sure if it's an xbox only thing. Can you check if this happens when you use a ps4 controller?

Reply:
Good catch, I doubt it's controller specific. I think for the sake of time I'll just add the same checks here for controller as we did above

Reply:
Oki doki. Let me know when it's implemented so I could test it :)